src/backend/OpenCL/CHIPBackendOpenCL.cc

```diff
@@ -1457,7 +1473,7 @@ chipstar::EventMonitor *CHIPBackendOpenCL::createStaleEventMonitor_() {
 }
 
 std::string CHIPBackendOpenCL::getDefaultJitFlags() {
-  return std::string("-x spir -cl-kernel-arg-info");
+  return std::string("-x spir -cl-kernel-arg-info -cl-std=CL3.0");
```

Comment:
Should update README.md to state the OpenCL 3.0 is the minimum driver version required? Currently, it states OpenCL 2.0 is the minimum required version.

Reply:
Actually, I'll try if it works with CL2.0 still.

Reply:
No, actually it's best use CL3.0 since it's likely more wide spread (in the future) than CL2.0.